src/core/audiosource.cpp

```diff
@@ -244,6 +263,14 @@ void FFMS_AudioSource::ResampleAndCache(CacheIterator pos) {
 		OutPlanes, new_req, DecodeFrame->nb_samples,
 		DecodeFrame->extended_data, DecodeFrame->nb_samples * av_get_bytes_per_sample(CodecContext->sample_fmt), DecodeFrame->nb_samples);
 #else
+#if defined(WITH_SWRESAMPLE)
```

Comment:
This is *still* incorrect.

When you build FFmpeg with MSVC, as a static library, it is *named* `.a`, but it is indeed a MSVC lib.

Seriously. Try it. Don't want to? Here's a video: https://vimeo.com/69111871 (a little dated; you don't know `c99conv` anymore, but it shows the output.) 

The `.lib` files are **only** produced with shared builds of FFmpeg, which generates import libraries during link. The MinGW build also includes said `.lib` import libraries. How do I know? **I am one of the people which added MSVC support to FFmpeg and its configure in the first place.**

To reiterate, static builds with MSVC produce `.a` files. They are not `ar` archives. They are MSVC archives. The `.a` is a name only.